.github/workflows/docker.yml

```diff
@@ -0,0 +1,20 @@
+name: Docker
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Build and Push to Registry
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: geostreams/geostreams-api-v3
```

Comment:
Can we start renaming this to `geostreams/geostreams` or do we want to do it later?

Reply:
Done.